lr_face/models.py

```diff
@@ -84,12 +81,18 @@ def predict_proba(self, X: List[FacePair]) -> np.ndarray:
         :return np.ndarray
         """
         scores = []
+        rm_pair = []
         cache_dir = EMBEDDINGS_DIR
         for pair in X:
             embedding1 = self.embedding_model.embed(pair.first, cache_dir)
             embedding2 = self.embedding_model.embed(pair.second, cache_dir)
-            score = spatial.distance.cosine(embedding1, embedding2)
-            scores.append([score, 1 - score])
+            if embedding1 is not None and embedding2 is not None:
+                score = spatial.distance.cosine(embedding1, embedding2)
+                scores.append([score, 1 - score])                
+            else:
+                rm_pair.append(pair)  # Remove pairs in which face is not detected
+        X = [i for i in X if i not in rm_pair] 
```

Comment:
this doesn't seem like a very tidy solution. isnt this part of another pull request?